### PR TITLE
Download page

### PR DESCRIPTION
**Pull request by fabiocaruso**

### CODE REVIEW

File: src/main/java/net/adoptium/DownloadResource.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+import net.adoptopenjdk.api.v3.models.Binary;
+
+@Path("/download")
+public class DownloadResource {
+    private static final Logger LOG = Logger.getLogger(DownloadResource.class);
+
+    @CheckedTemplate
+    public static class Templates {
+        public static native TemplateInstance download(DownloadResourceHTMLData htmlData);
```
Comment (hendrikebbers) [inline src/main/java/net/adoptium/DownloadResource.java:28]: Can you please add some doc to this class & native method

File: src/main/java/net/adoptium/UserSystem.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+public class UserSystem {
+    private OperatingSystem os;
+    private Architecture arch;
+
+    public UserSystem() {
+
+    }
+
+    public UserSystem(OperatingSystem os, Architecture arch) {
```
Comment (hendrikebbers) [inline src/main/java/net/adoptium/UserSystem.java:14]: Do we really need several constructors and setters?

File: src/main/resources/templates/DownloadResource/download.html [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+        <p>
+            If the downlaod does not start within 10 seconds,<br>
+            <a href="{htmlData.getDownloadLink()}">try again to download manually</a>
+        </p>
+        <p>checksum (sha256): {htmlData.getChecksum()}</p>
+        <button id="btn-copy-chksm">Copy</button>
+    </div>
+</div>
+<style type="text/css" scoped>
+    h1, h2, h3, h4, h5, h6 {
```
Comment (hendrikebbers) [inline src/main/resources/templates/DownloadResource/download.html:31]: Can we externalise CSS in a separate file?

**Review by hendrikebbers — APPROVED** [review]
minor suggestions

File: src/main/java/net/adoptium/DownloadResource.java (hunk 1 above)
Comment (thomasbuechi) [reply src/main/java/net/adoptium/DownloadResource.java:37]: added javadoc

File: src/main/java/net/adoptium/UserSystem.java (hunk 2 above)
Comment (thomasbuechi) [reply src/main/java/net/adoptium/UserSystem.java:10]: removed unused Constructor

File: src/main/resources/templates/DownloadResource/download.html (hunk 3 above)
Comment (thomasbuechi) [reply src/main/resources/templates/DownloadResource/download.html:31]: CSS externalized in own file index.css

**Pull request merged**